### ISSUE TITLE
Additional Lines on slide while opening specific PPTX

### ISSUE THREAD

**Issue by shu0406** [ONLYOFFICE/DocumentServer#1116]
**Do you want to request a *feature* or report a *bug*?**
bug = missing feature
**What is the current behavior?**
there is no possibility to add different designs to different slides in a presentation (missing feature). If you open a MS pptx with different designs in different slides, they all get mixed up in all slides (bug)
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
same bug in editor v 6.1.0.90 and docs 6.2.0 on windows 10 
**Did this work in previous versions of DocumentServer?**
**DocumentServer version:**
6.2.0
**Operating System:**
win10
**Browser version:**
brave/firefox

**Comment by Rita-Bubnova** [comment]
Hello, @shu0406.
What do you mean "different designs"? 
Are you talking about "Themes"?

**Comment by shu0406** [comment]
Yes, I was talking about templates. So if you have a horizontal bar in one slide and a vertical bar in the other slide in MS Powerpoint they get mixed up in ONLYOFFICE and you have both horizontal and vertical bars on every slide.

**Comment by Rita-Bubnova** [comment]
You can change the theme for individual slides. You need: select the slide -> open the context menu (right-click) -> change theme. More information [here](https://helpcenter.onlyoffice.com/en/onlyoffice-editors/onlyoffice-presentation-editor/usageinstructions/setslideparameters.aspx#).

But you seem to be having a problem with the specific presentation? Could you attach a file with the problem?

**Comment by shu0406** [comment]
Thanks, I did not know that option.

However then it seems to be a problem with a MS office presentation. I 
attached the file of one of my students. I checked it on ms office, it 
should be different themes on every slide. For example on the first page 
should be plain text, no bars at all. In the second slide there should 
be just the grey bars and in the third just one black bar. In onlyoffice 
it is just a "mix" of all themes...


Am 29.12.2020 um 10:59 schrieb Rita-Bubnova:

**Comment by Rita-Bubnova** [comment]
@shu0406, sorry, I don't see the attached file.

**Comment by shu0406** [comment]
I attached it to the e-mail I sent. Here again:
[problem with themes.pptx](https://github.com/ONLYOFFICE/DocumentServer/files/5750615/problem.with.themes.pptx)

**Comment by Rita-Bubnova** [comment]
Thank you for attached file.
I can confirm - this is bug, issue 48176 in our private issue tracker.

**Comment by Rita-Bubnova** [comment]
Sorry, we forgot to mention that issue was fixed.
I close this issue. Feel free to comment or reopen it if you got further questions.
